docs/specs/git.yml

```diff
@@ -406,7 +406,7 @@ repos:
 outputs:
   .errors.log: |
     ["error","committish-not-found","Can't find branch, tag, or commit 'live' for repo *"]
-    ["error","committish-not-found","Can't find branch, tag, or commit 'live' for repo *","a.md"]
+    ["error","committish-not-found","Can't find branch, tag, or commit 'live' for repo *"]
```

Comment:
why file info is missing for build error?

Reply:
It is a global error and was mistakenly reported as per file error due to we automatically append file info before